docs/content/installation/uninstall_driver_github.md

```diff
@@ -2,17 +2,36 @@
 
 Use this information to uninstall the IBM® CSI (Container Storage Interface) operator and driver with GitHub.
 
-Perform the following steps in order to uninstall the CSI driver and operator.
+Perform the following steps to uninstall the CSI driver and operator.
+
+**Note:** When using host definition and `dynamicNodeLabeling` is set to `true`, if these steps is not completed in the correct order, `hostdefiner.block.csi.ibm.com/manage-node=true` labels can be left on the nodes.
+
 1. Delete the IBMBlockCSI custom resource.
 
     ```
     kubectl delete -f csi.ibm.com_v1_ibmblockcsi_cr.yaml
     ```
 
-2. Delete the operator.
+2. If applicable, delete the HostDefiner custom resource.
+
+    1. Verify that all host definition instances of the node are deleted.
+     
+            kubectl get hostdefinition | grep <node-name> | wc -l
+     
+        The output should be `0`.
```

Comment:
what if `allowDelete` is `false`?

Reply:
@matancarmeli7 ? @roysahar ? any special information to write here if it's false?

Reply:
so maybe we can remove the `| wc -l ` and say that the user should know which hostDefinitions should be deleted and wait that those hostDefinitions were deleted

Reply:
@matancarmeli7 can i leave in the exact same step and just remove the `|wc -l`?

Reply:
yes I think so

Reply:
>yes I think so

@matancarmeli7 I'm pretty sure the output should **not** be `0` (or any number for that matter) once removing the command that outputs a count (`wc`)

please sync with @rivkap about this properly

Reply:
I thought she meant that she will remove the `0` also

Reply:
I just checked the change now and saw it, @rivkap the output shouldn't be `0`, it should be all the `hostdefinitions` that don't need to be deleted (if all of them should be deleted so we need to get `No resources found`)

Reply:
I meant that more thought should be given to this step in general. for example:
>of the node

which node?

>Verify

how does this relate to the `allowDelete` being `false`?
should the user manually delete `hostdefinition` resources? is it mandatory or optional?

Reply:
updated here
note that the wc -l is also found in [Changing node connectivity](https://github.com/IBM/ibm-block-csi-driver/blob/docs_release-1.10.0/docs/content/using/changing_node_connectivity.md) and in [Using host definition](https://github.com/IBM/ibm-block-csi-driver/blob/docs_release-1.10.0/docs/content/using/using_hostdefinition.md)

should these be changed as well or leave there as that is different circumstances?

Reply:
@matancarmeli7 would:
`kubectl get hostdefinitions -o=jsonpath='{range .items[?(@.spec.hostDefinition.nodeName=="<ndoe-name>")]}{.metadata.name}{"\n"}{end}'`
be better than `grep`?

Reply:
updated during meeting. possible different command - depending on @matancarmeli7  @oriyarde 
FYI @roysahar ?
thanks

Reply:
@rivkap, what should be the output of @matancarmeli7 suggested command? I guess "no output" and not "0" as. with my command, so this also should be changed.

Reply:
@roysahar following a meeting with matan and ori the other day we removed the wc -l and so they said that it was a different output. 
Ori wasn't super happy with that command in any case and suggested a different command that is more defined from our point of view ([as above](https://github.com/IBM/ibm-block-csi-driver/pull/532#discussion_r925339428))
Yes - we would need the updated output for this new command.
Is the command OK though? Is it what we need? and if so, should it be used in _all_ instances of where we currently have this command written?

Reply:
@rivkap I do not understand which command has been decided to use?

Reply:
@roysahar  that's part of the question. discussing separately and will summarize here

Reply:
updated per slack discussion